keepassxc-browser/_locales/en/messages.json

```diff
@@ -0,0 +1,742 @@
+    },
+    "optionsColumnDelete": {
+        "message": "Delete",
+        "description": "Site preferences list column title."
+    },
+    "optionsSelectionNothing": {
+        "message": "Enable all features",
+        "description": "Site preferences option selection."
+    },
+    "optionsSelectionNormal": {
+        "message": "Disable new/modified credentials",
+        "description": "Site preferences option selection."
+    },
+    "optionsSelectionFull": {
+        "message": "Disable all features",
+        "description": "Site preferences option selection."
+    },
+    "optionsCustomFieldsNotFound": {
+        "message": "No saved custom credential fields found.",
+        "description": "Shown when no saved custom credentials are saved."
+    },
+    "optionsCustomFieldsTabHelpTextFirst": {
+        "message": "If KeePassXC-Browser detects the wrong credential fields, you are able to specify the correct fields by yourself.",
+        "description": "Saved custom credential fields info text, first part."
+    },
+    "optionsCustomFieldsTabHelpTextSecond": {
+        "message": "Go to the page and click on the KeePassXC-Browser icon, then select ",
+        "description": "Saved custom credential fields info text, second part."
+    },
+    "optionsCustomFieldsTabHelpTextThird": {
+        "message": "On this page you can manage saved custom credential fields.",
+        "description": "Saved custom credential fields info text, third part."
+    },
+    "optionsCustomFieldsRemove": {
+        "message": "Remove saved custom credential fields?",
+        "description": "Confirmation text when removing saved custom credential fields."
+    },
+    "optionsCustomFieldsConfirmation": {
+        "message": "Do you really want to remove the saved custom credential fields on the page: $1",
+        "description": "Confirmation text when removing saved custom credential fields."
+    },
+    "optionsCustomFieldsConfirmationHelpText": {
+        "message": "KeePassXC-Browser will automatically detect the credential fields next time you visit this page.",
+        "description": "Part of confirmation text when removing saved custom credential fields."
+    },
+    "optionsSitePreferencesNotFound": {
+        "message": "No ignored sites found.",
+        "description": "Shown when no ignored sites are saved."
+    },
+    "optionsSitePreferencesRemove": {
+        "message": "Remove site?",
+        "description": "Confirmation text when removing ignored sites from the list."
+    },
+    "optionsSitePreferencesTabHelpTextFirst": {
+        "message": "Sites on this page have special handling methods associated with them.",
+        "description": "Site preferences info text, first part."
+    },
+    "optionsSitePreferencesTabHelpTextSecond": {
+        "message": "To ignore new/modified credentials on a specific site, add it below or click the blinking KeePassXC-Browser icon and select",
```

Comment:
add them.

Again, this seems to be only a partial string.

Reply:
This is problematic one because the placeholder is another `data-i18n`. It doesn't get translated correctly.